DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/board_test/README.md

```diff
@@ -228,6 +230,7 @@ The complete board test is divided into six subtests. By default, all tests run.
 | 4            | Kernel Latency Measurement

```

Comment:
this comment is for lines 227-230:

you defined 5 metrics earlier in this document, but there are 7 tests here. please either describe the other 2 in the section above, or condense the tests :)